README.rst

```diff
@@ -1,115 +1,339 @@
-   kinds of ways to contribute to and interact with the project,
-   including mailing list information and how to submit bug reports and
-   code to Ansible.
-*  Join a `Working Group
-   <https://github.com/ansible/community/wiki>`_, an organized community devoted to a specific technology domain or platform.
-*  Submit a proposed code update through a pull request to the ``devel`` branch.
-*  Talk to us before making larger changes
-   to avoid duplicate efforts. This not only helps everyone
-   know what is going on, it also helps save time and effort if we decide
-   some changes are needed.
-*  For a list of email lists, IRC channels and Working Groups, see the
-   `Communication page <https://docs.ansible.com/ansible/latest/community/communication.html>`_
-
-Coding Guidelines
-=================
-
-We document our Coding Guidelines in the `Developer Guide <https://docs.ansible.com/ansible/devel/dev_guide/>`_. We particularly suggest you review:
-
-* `Contributing your module to Ansible <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_checklist.html>`_
-* `Conventions, tips and pitfalls <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html>`_
-
-Branch Info
-===========
-
-*  The ``devel`` branch corresponds to the release actively under development.
-*  The ``stable-2.X`` branches correspond to stable releases.
-*  Create a branch based on ``devel`` and set up a `dev environment <https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html#common-environment-setup>`_ if you want to open a PR.
-*  See the `Ansible release and maintenance <https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html>`_ page for information about active branches.
-
-Roadmap
-=======
+*****************************
+ThinkAgile CP Ansible Modules
+*****************************
+
+This repository contains Lenovo-created Ansible modules
+for automating workflows with the ThinkAgile CP composable cloud platform.
+
+Getting Started
+===============
+The following instructions will show you how to download and install the
+tacp_info, tacp_instance, and tacp_network modules into your existing 
+Ansible environment.
+
+Requirements
+------------
+- Python >= 3.6
+- python3-pip
+- ansible
+- tacp
+
+
+Installation
+------------
+Download this git respository.
+
+``cd ~``
+
+``git clone https://github.com/lenovo/ansible.lenovo-tacp.git``
+
+``cd ./ansible.lenovo-tacp``
+
+|
+
+The tacp modules and module_utils files can be added to any of the following dfirectories:
+
+- any directory added to the ``ANSIBLE_LIBRARY`` environment variable (``$ANSIBLE_LIBRARY`` takes a colon-separated list like ``$PATH``)
+
+- ``~/.ansible/plugins/``
+
+- On CentOS/RHEL systems, ``/usr/share/ansible/plugins/``
+
+|
+
+1. For example, create the ``~/.ansible/plugins/`` directory:
+
+``[user@hostname ansible.lenovo-tacp]# mkdir -p ~/.ansible/plugins/``
```

Comment:
Remove the `[user@hostname ansible.lenovo-tacp]#`

Convention is generally:
```bash
$ mkdir -p ~/.ansible/plugins/
```

Reply:
From the instructions below, you should also make sure `module_utils` is created.

```bash
$ mkdir -p ~/.ansible/plugins/module_utils ~/.ansible/plugins/modules/cloud
```